backend/chalice/upload_failures/app.py

```diff
@@ -0,0 +1,19 @@
+from chalice import Chalice
+import os
+import sys
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "chalicelib"))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from upload_failures.upload import delete_many_from_s3, update_dataset_processing_status_to_failed
+
+
+app = Chalice(app_name="upload_failures")
+
+
+@app.lambda_function()
```

Comment:
Does this have to be a lambda separate from the code that runs in the upload container?

Reply:
The idea is to catch errors in the container we can't predict since the container is running community R code that is unstable. We want to cleanup anything left over if the container fails.